### PR TITLE
[KubernetesConfiguration-Flux] Move KubernetesConfiguration-Flux to main

### PR DESCRIPTION
**Pull request by lijinpei2008**
## Description
 * Create Some cmdlets:
    * {Get}-AzKubernetesConfigFluxOperationStatus/AzK8sConfigFluxOperationStatus
    * {Get/New/Update/Remove}-AzKubernetesConfigurationFlux/AzK8sConfigurationFlux

Swagger : https://github.com/Azure/azure-rest-api-specs/tree/main/specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/stable/2022-03-01

Design: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1150

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/KubernetesConfiguration/Az.KubernetesConfiguration.psd1 [hunk 1]
```diff
@@ -113,8 +121,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Onboarded cmdlets ''New/Update/Get/Remove-AzKubernetesExtension'' and ''New/Get/Remove-AzKubernetesConfiguration''
-* Removed the plural form of parameter HelmOperatorChartValues, OperatorParameters and SshKnownHosts in ''New-AzKubernetesConfiguration'''
+        # ReleaseNotes = ''
```
Comment (VeryEarly) [inline src/KubernetesConfiguration/Az.KubernetesConfiguration.psd1:124]: please paste your changelog here

File: src/KubernetesConfiguration/ChangeLog.md [hunk 2]
```diff
@@ -18,6 +18,9 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+ * Create Some cmdlets:
```
Comment (VeryEarly) [inline src/KubernetesConfiguration/ChangeLog.md:21]:
```suggestion
 * Added cmdlets:
   - `Get-AzKubernetesConfigFluxOperationStatus`
   - `Get-AzKubernetesConfigurationFlux`
   - `New-AzKubernetesConfigurationFlux`
   - `Update-AzKubernetesConfigurationFlux`
   - `Remove-AzKubernetesConfigurationFlux`
   
```

File: src/KubernetesConfiguration/README.md [hunk 3]
```diff
@@ -47,15 +47,17 @@ In this directory, run AutoRest:
...
-  - $(repo)/specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/stable/2021-03-01/kubernetesconfiguration.json
-  - $(repo)/specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/stable/2021-09-01/extensions.json
+  - $(repo)/specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/stable/2022-03-01/extensions.json
+  - $(repo)/specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/stable/2022-03-01/fluxconfiguration.json
+  - $(repo)/specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/stable/2022-03-01/operations.json
+  - $(repo)/specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/stable/2022-03-01/kubernetesconfiguration.json
 
 title: KubernetesConfiguration
-module-version: 0.1.0
+module-version: 0.6.0
```
Comment (VeryEarly) [inline src/KubernetesConfiguration/README.md:60]:
```suggestion
module-version: 0.1.0
```

File: src/KubernetesConfiguration/Az.KubernetesConfiguration.psd1 [hunk 4]
```diff
@@ -121,7 +121,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        # ReleaseNotes = ''
+        ReleaseNotes = ' * Added cmdlets: * ''Get-AzKubernetesConfigFluxOperationStatus'' * ''Get-AzKubernetesConfigurationFlux'' * ''New-AzKubernetesConfigurationFlux'' * ''Update-AzKubernetesConfigurationFlux'' * ''Remove-AzKubernetesConfigurationFlux'''
```
Comment (VeryEarly) [inline src/KubernetesConfiguration/Az.KubernetesConfiguration.psd1:124]: please remove this part

Comment (lijinpei2008) [reply src/KubernetesConfiguration/Az.KubernetesConfiguration.psd1]: Done.

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**